2020/10/08 19:10:59 [error] 28#28: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: localhost, request: "GET /api/ HTTP/1.1", upstream: "http://172.18.0.4:3000/", host: "localhost"
